pkg/metrics/metrics.go

```diff
@@ -39,6 +38,7 @@ const (
 
 type metricsConfig struct {
 	ServerURL string `env:"ABC_METRICS_URL, default=https://abc-updater-metrics.tycho.joonix.net"`
+	NoMetrics bool   `env:"NO_METRICS"`
```

Comment:
Should this be `ABC_NO_METRICS` for consistency with `ABC_METRICS_URL`?

Reply:
I'm not second-guiessing myself.

I think it does make sense to namespace somehow.

Currently its done by `${APP_ID}_${METRIC_NAME}` so this becomes for `abc` `ABC_ABC_METRICS_URL`.

Maybe we should decide on a name for `abc-updater` such as `ABCU`. So it would be `ABC_ABCU_METRICS_URL` and `ABC_ABCU_IGNORE_VERSIONS`

Thoughts?

Reply:
Either that or not namespace at all, though you run the risk of collision with the app that is using `abc-updater` (though a custom lookupper could be passed in using `WithLookupper`

Reply:
Your "no namespace" option seems pretty clean; I'd be willing to accept the risk of collisions for a clean UX here

Reply:
Removed (note this is technically a breaking change for `ABC_METRICS_URL` in `abc`

Reply:
Cool, I very much doubt anyone is overriding ABC_METRICS_URL.